packages/storybook-utils/src/actions.ts

```diff
@@ -23,6 +23,40 @@ export const enhanceEventArgTypes: ArgTypesEnhancer = ({ argTypes }) => {
   return argTypes;
 };
 
+/**
+ * Allows logging and documentation for the passed event listener names in Storybook.
+ * Will be documented in a extra "Relevant HTML events" section in the Storybook documentation.
+ *
+ * @example
+ * ```typescript
+ * const meta: Meta<typeof OnyxButton> = {
+ *   title: "Buttons/Button",
+ *   component: OnyxButton,
+ *   argTypes: {
+ *     somethingElse: { ...someOtherArgType },
+ *     ...withNativeEventLogging(["onClick"]),
+ *  },
+ *};
+ * ```
+ *
+ * @param relevantEvents a list of event names that should be logged
+ * @returns Storybook ArgTypes object
+ */
+export const withNativeEventLogging = (relevantEvents: (keyof Events)[]) =>
+  relevantEvents.reduce((argTypes, eventName) => {
+    const action = eventName.replace(/^on/, "").toLowerCase();
+    argTypes[eventName] = {
+      name: eventName,
+      control: false,
+      description: `[${action}_event](https://developer.mozilla.org/en-US/search?q=element+${action}+event)`, // unfortunately there is no static path for all events
```

Comment:
(optional) we could maintain a map of "real" links for very common events like click, change, focus etc. and use the generic search link for all other events, what do you think?

If its too much manual maintenance we don't have to do it :)